actor/src/main/scala/org/apache/pekko/dispatch/Dispatcher.scala

```diff
@@ -64,16 +64,20 @@ class Dispatcher(
     new LazyExecutorServiceDelegate(executorServiceFactoryProvider.createExecutorServiceFactory(id, threadFactory))
 
   protected final def executorService: ExecutorServiceDelegate = executorServiceDelegate
+  protected final def isVirtualized: Boolean = executorServiceFactoryProvider.isVirtualized
 
   /**
    * INTERNAL API
    */
   protected[pekko] def dispatch(receiver: ActorCell, invocation: Envelope): Unit = {
     val mbox = receiver.mailbox
     mbox.enqueue(receiver.self, invocation)
-    registerForExecution(mbox, true, false)
+    registerForExecution(mbox, hasMessageHint = true, hasSystemMessageHint = false)
   }
 
+  final override def batchable(runnable: Runnable): Boolean =
+    if (isVirtualized) true else super.batchable(runnable)
```

Comment:
the PR title says 'disable batch if isVirtualized' but this seems to be the opposite

Reply:
Opps...